.claude/commands/cleanup-worktree.md

```
@@ -0,0 +1,120 @@
+argument-hint: <branch-slug, optional — omit to sweep ALL completed worktrees>
+---
+Clean up finished OloEngine worktrees now that their work is merged.
+
+## 0. Locate the base repo (derive it, don't assume a path)
+    git rev-parse --path-format=absolute --git-common-dir
+The parent of that .git directory is the BASE REPO; call it $BASE. Run every command
+below with `git -C $BASE ...`. If THIS session is running inside a worktree you're
+about to remove, stop and tell me to re-run this from a different window (the base repo
+or another worktree) — git can't remove the worktree you're standing in.
+
+## 1. Decide the target set
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)